tests/integration/targets/nxos_static_routes/tasks/cli.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Can you please update this to the content of https://raw.githubusercontent.com/ansible-collections/cisco.nxos/fc949445e6c4c91365c702ebf282c6394445011c/tests/integration/targets/nxos_lacp/tasks/cli.yaml?